### ISSUE TITLE
Fuck you Author

### ISSUE THREAD

**Issue by Cho1990-PSY** [bbaaii/DreamDiffusion#25]
Mother fuck this is totally real shit! whole code is hard coded fitted just their own data not working for other data.
even my dog is not coding like this. this fucking no mother code is really really shit and robbed other code just copy and paste all. fuck you and fuck you and fuck you and fuck you. this is humiliate all brain scientist and AI developmenter. if you don't know about stable diffusion and eeg and erp please study about all of them. and really don't know about the whole code. call your mother you real shit mother fucker. fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and fuck you and

**Comment by hu975110710** [comment]
6

**Comment by AlexxxStarkkk** [comment]
有人成功复现过吗

**Comment by DylanLIiii** [comment]
The pre-training results of the article are completely irreproducible. The paper lacks even the basic preprocessing steps for the data, and the datasets used in pre-training exhibit significant magnitude differences without any preprocessing! The seemingly good results of the study are solely due to the mandatory alignment in the final step on the Clip Encoder. While this approach undoubtedly yields positive outcomes, it prevents the model from generalizing beyond the local dataset. I am uncertain of the value of such work. If one were to treat EEG as a time series for training, I would recommend adopting methods from works like TiMAE instead.

**Comment by wangxinrongCSDN** [comment]
The preprocessing steps are completely irreducible, and this kind of article is also what my most previous blog said, that many of the articles' models are just slaves to their own data

**Comment by shuohantao** [comment]
I just don't get the part where padding unpreprocessed data with completely different number of channels collected from completely different devices to the same number of dimensions and expect the model to learn a shared representation is a good idea.

**Comment by f13T2ach** [comment]
interesting

**Comment by BSoDium** [comment]
Insightful 🤌

**Comment by AlexHughesOk** [comment]
illuminating

**Comment by putra4648** [comment]
Illusion 🤌

**Comment by yogipras-hyper** [comment]
Dafug

**Comment by Ry3yr** [comment]
Very mature indeed

**Comment by mahui-HUE** [comment]
我就喜欢你这直爽的性子。

**Comment by saxenanamay** [comment]
I totally agree, this is a real shit! 💩💩💩💩💩💩💩💩💩💩💩💩💩💩💩💩💩

**Comment by codecooker1** [comment]
Dafaq

**Comment by lll143653** [comment]
agree

**Comment by hjYu-711** [comment]
This paper is completely meaningless, just a piece of shit. Its high-quality images come from the prior knowledge of the diffusion model, and the forced alignment by CLIP causes the model to have no generalization ability.

**Comment by YuRongHao2233** [comment]
> 这篇论文完全没有意义，只是一坨狗屎。其高质量的图像来自扩散模型的先验知识，CLIP的强制对齐导致模型没有泛化能力

貌似目前的sota，GWIT也是这个样子

**Comment by vexpaer** [comment]
你是谁> lol

**Comment by Labhahaha** [comment]
Essentially, this article only completed a in-domain classification task of EEG signals within 40 categories through CLIP alignment.

**Comment by Le-xiaoyu** [comment]
i like your speaking style!!!

**Comment by Arandinglv** [comment]
Love from Redbook

**Comment by sunningilyA7** [comment]
hahhhhh
